docs/pages/kubernetes-access/helm/reference.mdx

```diff
@@ -0,0 +1,1788 @@
+---
+title: Helm chart reference
+description: A list of values that can be set using Teleport Helm charts and their purpose
+---
+
+The Teleport Helm repository hosts two Helm charts:
+
+- [`teleport-cluster`](#teleport-cluster) [(source on Github)](https://github.com/gravitational/teleport/tree/master/examples/chart/teleport-cluster)
+- [`teleport-kube-agent`](#teleport-kube-agent) [(source on Github)](https://github.com/gravitational/teleport/tree/master/examples/chart/teleport-kube-agent)
+
+## `teleport-cluster`
+
+The `teleport-cluster` Helm chart is used to configure a Teleport cluster. It runs two Teleport services:
+
+| Teleport service | Purpose | Documentation |
+| - | - | - |
+| `auth_service` | Authenticates users and hosts, and issues certificates | [Auth documentation](../../architecture/authentication.mdx)
+| `proxy_service`| Runs the externally-facing parts of a Teleport cluster, such as the web UI, SSH proxy and reverse tunnel service | [Proxy documentation](../../architecture/proxy.mdx) |
+
+The `teleport-cluster` chart can be deployed in four different modes. We provide a detailed guide for each mode:
+
+| `chartMode` | Guide |
+| - | - |
+| `standalone` | [Getting started with Kubernetes Access](../getting-started.mdx) |
+| `aws` | [Running an HA Teleport cluster using an AWS EKS Cluster](./guides/aws.mdx) |
+| `gcp` | [Running an HA Teleport cluster using a Google Cloud GKE cluster](./guides/gcp.mdx) |
+| `custom` | [Running a Teleport cluster with a custom config](./guides/custom.mdx) |
+
+This reference details available values for the `teleport-cluster` chart.
+
+## `clusterName`
```

Comment:
A multi-level TOC which uses the level of the header comments would be really nice, or some other way to auto-generate a TOC at the top of the file rather than using the side nav.